tests/storage/__init__.py

```diff
@@ -249,8 +250,8 @@ def test_specialchars(self, monkeypatch, requires_collections,
                           get_storage_args, get_item):
         if getattr(self, 'dav_server', '') == 'radicale':
             pytest.skip('Radicale is fundamentally broken.')
-        if getattr(self, 'dav_server', '') == 'icloud':
-            pytest.skip('iCloud rejects uploads.')
+        if getattr(self, 'dav_server', '') in ('icloud', 'fastmail'):
+            pytest.skip('iCloud and FastMail reject this name.')
```

Comment:
It would be totally understandable if they intentionally reject this upload, it amounts to fuzzing what I do here.